src/scripts/validate.js

```diff
@@ -18,6 +18,9 @@ const useDefaultScripts = typeof validateScripts !== 'string'
 const scripts = useDefaultScripts
   ? {
       build: ifScript('build', 'npm run build --silent'),
+      format: preCommit
+        ? null
+        : ifScript('format', 'npm run format -- --check'),
```

Comment:
What happens when prettier is passed `--write` and `--check` at the same time?

Also, this kinda assumes that the `format` script is using something that accepts the `--check` flag (like `kcd-scripts format` which forwards the flags to prettier). So it probably does, but maybe we should note that in a code comment.

Reply:
The `test` one also assumes there's a `test` script that accepts a `--coverage` flag, so I don't think we should mind the `--check` flag

Have to look into the code so we don't send the `--write` flag when checking I think.